blogger/blogger.go

```diff
@@ -1,4 +1,4 @@
-package cassiopeia
+package blogger
```

Comment:
I think it should be named as `blog` instead of `blogger`: package's name `blog`, interface's name `blogger`. To be consistent with `hall_of_fame`, which interface is `hall_of_famer`.